plugins/database/quota/src/org/apache/cloudstack/api/command/QuotaSummaryCmd.java

```diff
@@ -59,7 +59,7 @@ public QuotaSummaryCmd() {
     public void execute() {
         Account caller = CallContext.current().getCallingAccount();
         List<QuotaSummaryResponse> responses;
-        if (caller.getAccountId() <= 2) { //non root admin or system
+        if (caller.getAccountId() <= 2) { // root admin or system
```

Comment:
to assume that an account <=2 has root admin privileges is just naive. There must be a solid way to do this. Just my 2 cents..